lib/newman_conway.py

```diff
@@ -4,7 +4,24 @@
 # Space Complexity: ?
 def newman_conway(num):
     """ Returns a list of the Newman Conway numbers for the given value.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n) ? 
```

Comment:
✨ Correct on the time complexity. You have two consecutive for loops, each of which iterates approximately n times; 3...n is about the same as 1...n as n gets large